content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -362,6 +362,40 @@ For more information on inter-pod affinity/anti-affinity, see the
 You may want to check [Taints](/docs/concepts/configuration/taint-and-toleration/)
 as well, which allow a *node* to *repel* a set of pods.
 
+## nodeName
+
+`nodeName` is the simplest form of node selection constraint, but due
+to its limitations it is typically not used.  `nodeName` is a field of
+PodSpec.  If it is non-empty, the scheduler ignores the pod and the
+kubelet running on the named node tries to run the pod.  Thus, if
+`nodeName` is provided in the PodSpec, it takes precedence over the
+above methods for node selection.
+
+Some of the limitations of using `nodeName` to select nodes are:
+
+-   If the named node does not exist, the pod will not be created.
```

Comment:
suggest "If the named node does not exist, the pod will not be run, and in some cases may be automatically deleted."

Reply:
Changed